app/src/components/styled/generic/StyledHeadings.ts

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
```

Comment:
Not sure about this folder structure, I would just rename it to Headings.ts and put it in "components"

Reply:
I'd like to introduce at least a "styled" folder as there are quite a few components that aren't styled and so that folder is getting quite full

Reply:
@Jbithell You have done larger react projects before haven't you, what have you done in that past for the file structure of components? Just wondering really

Reply:
Have combined into one file